custom-docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Set default to invalid version to ensure override in build.sh
+ARG TRINO_VERSION=472
+FROM trinodb/trino-core:${TRINO_VERSION}
+
+# Add customization
+COPY --chown=trino:trino target/customization /
+RUN rm -rf /usr/lib/trino/plugin/geospatial \
```

Comment:
Why is this needed? Add a code comment. I thought the core image has the smallest possible set of plugins, none of which are really needed to be removed.

Reply:
Its not needed. Just an example. I commented it out and added details about what it does in a comment above so this should be good. I adjusted the readme as well.